### ISSUE TITLE
1.1.6 - Undefined variable: imgClass

### ISSUE THREAD

**Issue by Aleksey-Maksimov** [Mikesch-mp/icingaweb2-module-grafana#68]
Hi.

Just updated to 1.1.6. 
Host and Service page show error:

```
#0 /usr/share/icingaweb2/modules/grafana/library/Grafana/ProvidedHook/Grapher.php(254): Icinga\Application\ApplicationBootstrap->Icinga\Application\{closure}(8, 'Undefined varia...', '/usr/share/icin...', 254, Array)
#1 /usr/share/icingaweb2/modules/grafana/library/Grafana/ProvidedHook/Grapher.php(357): Icinga\Module\Grafana\ProvidedHook\Grapher->getMyPreviewHtml('hostalive4', 'KOM-APP01...', '')
#2 zend.view:///usr/share/icingaweb2/modules/monitoring/application/views/scripts/show/components/grapher.phtml(2): Icinga\Module\Grafana\ProvidedHook\Grapher->getPreviewHtml(Object(Icinga\Module\Monitoring\Object\Host))
#3 /usr/share/php/Icinga/Web/View.php(231): include('zend.view:///us...')
#4 /usr/share/icingaweb2/library/vendor/Zend/View/Abstract.php(877): Icinga\Web\View->_run('/usr/share/icin...')
#5 zend.view:///usr/share/icingaweb2/modules/monitoring/application/views/scripts/partials/object/detail-content.phtml(3): Zend_View_Abstract->render('show/components...')
#6 /usr/share/php/Icinga/Web/View.php(231): include('zend.view:///us...')
#7 /usr/share/icingaweb2/library/vendor/Zend/View/Abstract.php(877): Icinga\Web\View->_run('/usr/share/icin...')
#8 zend.view:///usr/share/icingaweb2/modules/monitoring/application/views/scripts/host/show.phtml(14): Zend_View_Abstract->render('partials/object...')
#9 /usr/share/php/Icinga/Web/View.php(231): include('zend.view:///us...')
#10 /usr/share/icingaweb2/library/vendor/Zend/View/Abstract.php(877): Icinga\Web\View->_run('/usr/share/icin...')
#11 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action/Helper/ViewRenderer.php(904): Zend_View_Abstract->render('host/show.phtml')
#12 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action/Helper/ViewRenderer.php(925): Zend_Controller_Action_Helper_ViewRenderer->renderScript('host/show.phtml', NULL)
#13 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action/Helper/ViewRenderer.php(964): Zend_Controller_Action_Helper_ViewRenderer->render()
#14 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action/HelperBroker.php(272): Zend_Controller_Action_Helper_ViewRenderer->postDispatch()
#15 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action.php(518): Zend_Controller_Action_HelperBroker->notifyPostDispatch()
#16 /usr/share/php/Icinga/Web/Controller/Dispatcher.php(76): Zend_Controller_Action->dispatch('showAction')
#17 /usr/share/icingaweb2/library/vendor/Zend/Controller/Front.php(937): Icinga\Web\Controller\Dispatcher->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response))
#18 /usr/share/php/Icinga/Application/Web.php(389): Zend_Controller_Front->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response))
#19 /usr/share/php/Icinga/Application/webrouter.php(109): Icinga\Application\Web->dispatch()
#20 /usr/share/icingaweb2/public/index.php(4): require_once('/usr/share/php/...')
#21 {main}
```

**Comment by Mikesch-mp** [comment]
Fixed it, please redownload 1.1.6

**Comment by dnsmichi** [comment]
Next time, please create a new version and release 1.1.7. This will prevent unnecessary support questions, and also allows you to just go for git tags. You don't do any atm, but it would be good to have them :)
